Library/Formula/storm.rb

```diff
@@ -1,8 +1,8 @@
 class Storm < Formula
   desc "Distributed realtime computation system to process data streams"
   homepage "https://storm.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=storm/apache-storm-0.9.4/apache-storm-0.9.4.tar.gz"
-  sha256 "2cdb3530839375bb5f18751ff16ff78b11e9b4f6d23c423c95d78487ad92d38e"
+  url "https://github.com/apache/storm/archive/v0.9.5.tar.gz"
```

Comment:
Let's keep the `apache.org/dyn` link here - It's still the link used on the official download page for Current Releases.